packages/core/test/emulate/mockXhr.ts

```diff
@@ -49,11 +49,12 @@ class MockEventEmitter {
     this.listeners[name] = this.listeners[name].filter((listener) => listener !== callback)
   }
 
-  protected dispatchEvent(name: string) {
-    if (!this.listeners[name]) {
+  dispatchEvent(evt: Event) {
```

Comment:
`dispatchEvent` is normally exposed on the native xhr, so I see no reason to not expose it for mockXHR, especially as I a change the signature to be more standard. 

It's just a tiny bit more verbose where dispatchEvent is called